src/MySQL.php

```diff
@@ -188,88 +288,116 @@ public function updateEntity(\obo\Entity $entity) {
 
             foreach ($entities as $entity) {
                 if (!$entity instanceof \obo\Entity) throw new \obo\Exceptions\Exception("Relationship can't be created. Entities must be of \obo\Entity instance");
             }
         }
 
-        $this->dibiConnection->query("INSERT INTO [{$repositoryName}] ", [$entities[0]->entityInformation()->repositoryName => $entities[0]->primaryPropertyValue(), $entities[1]->entityInformation()->repositoryName => $entities[1]->primaryPropertyValue()]);
+        $entityInformation = $entity->entityInformation();
+        $informationForEntity = $this->informationForEntity($entityInformation);
+        $this->dibiConnection->query("INSERT INTO [{$storageName}].[{$repositoryName}] ", [$entities[0]->entityInformation()->repositoryName => $entities[0]->primaryPropertyValue(), $entities[1]->entityInformation()->repositoryName => $entities[1]->primaryPropertyValue()]);
     }
 
+
     /**
-     * @param string $repositoryName
+     * @param string $repository
      * @param array $entities
      * @return void
      * @throws \obo\Exceptions\Exception
      */
-    public function removeRelationshipBetweenEntities($repositoryName, array $entities) {
+    public function removeRelationshipBetweenEntities($repository, array $entities) {
+        $repositoryStorageName = $this->extractStorageName($repository);
+        $repositoryRepositoryName = $this->extractRepositoryName($repository);
 
         if (\obo\obo::$developerMode) {
-            if (!$this->existsRepositoryWithName($repositoryName)) throw new \obo\Exceptions\Exception("Relationship can't deleted repository with the name '{$repositoryName}' does not exist");
+            if (!$this->existsRepositoryWithName($repositoryStorageName, $repositoryRepositoryName)) throw new \obo\Exceptions\Exception("Relationship can't deleted repository with the name '{$repositoryRepositoryName}' located in storage with name '{$repositoryStorageName}' does not exist");
             if (\count($entities) !== 2) throw new \obo\Exceptions\Exception("Relationship can't be deleted. Two entities were expected but " . \count($entities) . " given. ");
 
             foreach ($entities as $entity) {
                 if (!$entity instanceof \obo\Entity) throw new \obo\Exceptions\Exception("Relationship can't be deleted. Entities must be of \obo\Entity instance.");
             }
         }
 
-        $this->dibiConnection->query("DELETE FROM [{$repositoryName}] WHERE [{$entities[0]->entityInformation()->repositoryName}] = {$entities[0]->primaryPropertyValue()} AND [{$entities[1]->entityInformation()->repositoryName}] = " . $this->informationForEntity($entities[1]->entityInformation())["columns"][$entities[1]->entityInformation()->informationForPropertyWithName($entities[1]->entityInformation()->primaryPropertyName)->columnName]["placeholder"], $entities[1]->primaryPropertyValue());
+        $entities1StorageName = $this->getStorageName($entities[1]->entityInformation());
```

Comment:
It's not necessary to assign statement to a new variable. Variable is used only one time.